src/components/Filters/Filters.tsx

```diff
@@ -287,6 +287,7 @@ class FiltersInner extends Component<CombinedProps, State> {
             clearButton
             onClearButtonClick={onQueryClear}
             disabled={disabled}
+            autoComplete="off"
```

Comment:
every where else in this PR autoComplete="nope" but here it is "off" not sure if this is important but just wanted to bring it up.